### PR TITLE
Fix <title> tags in admin interface

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Fix <title> tags in admin interface
2. Rename variable names from `ractive` to component names in boxoffice admin

### CODE REVIEW

File: boxoffice/static/js/models/util.js [hunk 1]
```diff
@@ -21,23 +21,31 @@ export const Util = {
...
 }
 
-export const scrollToElement = function(element, speed=500) {
+export const scrollToElement = function (element, speed=500) {
   $('html,body').animate({
     scrollTop: $(element).offset().top
   }, speed);
 }
+
+export const completePageLoad = function (newTitle) {
```
Comment (shreyas-satish) [inline boxoffice/static/js/models/util.js]: Can you please add a comment to describe what this function does? An example would help.

**Comment by shreyas-satish** [comment]
The `<title>` tags are still showing up as 'HasGeek - Boxoffice' like before.

**Comment by vidya-ram** [comment]
@shreyas-satish Are you seeing this on item collection dashboard page?

**Comment by shreyas-satish** [comment]
@vidya-ram I'm seeing `<org> - Boxoffice` on the `/admin/o/<org>` page, and `<item collection title> - Boxoffice` on the dashboard, reports, orders pages.

**Comment by vidya-ram** [comment]
@shreyas-satish Yes, that was the intention. Is this behaviour fine?
1. `Admin - Boxoffice` on `admin` page
2. `<org title> - Boxoffice` on `/admin/o/<org>` page
3. `<item collection title> - Boxoffice` on the dashboard, reports, orders pages of item collection pages.

File: boxoffice/static/js/models/util.js [hunk 2]
```diff
@@ -21,23 +21,39 @@ export const Util = {
...
 
-export const scrollToElement = function(element, speed=500) {
+export const scrollToElement = function (element, speed=500) {
   $('html,body').animate({
     scrollTop: $(element).offset().top
   }, speed);
 }
+
+export const makePageTitle = function (title, subTitle) {
+  /* Takes title, subTitle and join them into a string using " — ".
```
Comment (shreyas-satish) [inline boxoffice/static/js/models/util.js]: "Takes a `title` and a `sub-title` and returns a concatenated string separated by " - "".

File: boxoffice/static/js/views/main.js [hunk 3]
```diff
@@ -13,4 +13,14 @@ $(function(){
   window.eventBus.on('navigate', function(msg){
     appRouter.navigate(msg, {trigger: true});
   })
+  window.admin = {};
+  /* Title tag in Baseframe template is defined as "pageTitle — siteTitle"
+  Split the page title to get the site title and assign it to window.admin.siteTitle
+  eg:- The page title is "JSFoo 2017 — Boxoffice", splits into ["JSFoo 2016", Boxoffice"]
+  where the last item is the site title */
+  window.admin.siteTitle = (function () {
+	  let currentPageTitle = $('title').html();
```
Comment (shreyas-satish) [inline boxoffice/static/js/views/main.js]: Indentation seems off here.

File: boxoffice/static/js/views/main.js [hunk 4]
```diff
@@ -13,4 +13,14 @@ $(function(){
   window.eventBus.on('navigate', function(msg){
     appRouter.navigate(msg, {trigger: true});
   })
+  window.admin = {};
+  /* Title tag in Baseframe template is defined as "pageTitle — siteTitle"
+  Split the page title to get the site title and assign it to window.admin.siteTitle
+  eg:- The page title is "JSFoo 2017 — Boxoffice", splits into ["JSFoo 2016", Boxoffice"]
+  where the last item is the site title */
+  window.admin.siteTitle = (function () {
```
Comment (shreyas-satish) [inline boxoffice/static/js/views/main.js]: Why is this defined on `window`?

**Pull request merged**